portfolio/src/main/webapp/index.html

```diff
@@ -4,44 +4,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Shandy Kim's Portfolio</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Maven+Pro:wght@500&display=swap">
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
   </head>
   <body>
-    <div id="content" class="container">
-      <div class="container p-3 my-3 bg-dark text-white">
+    <div id="content" class="container-fluid -fluid ">
+      <div class="container-fluid -fluid  p-3 my-3 bg-dark text-white">
```

Comment:
extra -fluid?

Reply:
I used a replace all and didn't notice some errors. Fixed!